### DISCUSSION TITLE
Update

### DISCUSSION THREAD

**Discussion by wartungsmonteur** [wills106/homeassistant-solax-modbus#475] [Q&A]
Hallo

Ich weiss leider nicht wie ich die jetzigen Daten im HA updaten kann, Die erste Verison habe ich runter laden können und per Samba auf den HA kopiert. Leider geht die nun nicht mehr. Ich bitte vielmals um Hilfe.

Viele Grüße und Dankeschön

**Comment by wills106** [comment]
The easiest option is to install https://hacs.xyz/ then you can update through the Home Assistant interface.

**Comment by wartungsmonteur** [comment]
Guten Morgen

Ich habe das alles so gemacht, leider geht das Addon nun nicht mehr. Ich habe gestern einen neuen Wechselrichter bekommen und bekomme nun keine Verbindung mehr über den Modbus. Ich habe den Logfile bereits hier hochgeladen. Scheinbar kommt das Addon mit der Seriennummer des WR nicht klar.

Vielen Dank für die Hilfe

**Comment by wartungsmonteur** [comment]
HalloEs gibt nach einem Update des Solax Wechselrichters einen neuen Modus.Zu finden unter Einstellungen / Betriebsmodus.Nennt sich:  Peak shavingNutzbar für flexible Stromtarife um diese  zeitgesteuert zu nutzen um z. B. den Akku aus dem Netz zu ladenMit freundlichen Grüßen .Heizung-Kälte-Klima-LüftungEnergieeffizienz HandwerkerLufthygiene TechnikerThermografieHeizungsservice Michael ZeißKurze Gasse 901896 Ohorn Tel.01723760357Fax 035955 579144
-------- Ursprüngliche Nachricht --------Von: wills106 ***@***.***> Datum: 25.07.23  22:17  (GMT+01:00) An: wills106/homeassistant-solax-modbus ***@***.***> Cc: wartungsmonteur ***@***.***>, Author ***@***.***> Betreff: Re: [wills106/homeassistant-solax-modbus] Update (Discussion #475) 
The easiest option is to install https://hacs.xyz/ then you can update through the Home Assistant interface.

**Comment by wartungsmonteur** [comment]
Alle PeakShaving Optionen nun verfügbar.Vielen Dank.
